tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -501,11 +501,8 @@ func TestRateLimit(t *testing.T) {
 			Duration:   1 * time.Minute,
 			NumThreads: 8,
 		},
-		HTTPOptions: fhttp.HTTPOptions{
-			URL: url,
-		},
 	}
-
+	opts.Init(url)
```

Comment:
Can we change this to `HTTPOptions: with the value of timeout passed in?`

Reply:
We can remove it after the other pr

Let's get this surgery first and we can cleanup the tests that use fortio later to use for instance the new "Exactly: 100" etc